.github/workflows/cmake.yml

```diff
@@ -1,128 +1,77 @@
             name: Linux arm64
-            install: $common_ubuntu_deps crossbuild-essential-arm64
-            # build current openssl libs instead of using old versions provided by apt on ubuntu 18
-            cmake_opts: "`-DVCPKG_OVERLAY_PORTS=./vcpkg-overlays/linux-syslibs/ubuntu18`"
-            target: linux-arm64
+            preset: linux-arm64
 
     steps:
-      - name: install tools
-        if: ${{ matrix.install != null }}
-        run: |
-          apt -y update
-          apt -y install ${{ matrix.install }}
-
-      - name: install contemporary git
-        if: ${{ matrix.container != null }}
-        run: |
-          apt -y update
-          apt -y install software-properties-common
-          add-apt-repository -y ppa:git-core/ppa
-          apt -y update
-          apt -y install git
-          git config --global --add safe.directory $GITHUB_WORKSPACE
-          git --version
-
-      # run this step if Linux and arm or arm64 
-      - name: subscribe to apt multiarch ports and install libssl-dev for target arch
-        if: matrix.container != null && matrix.target != 'linux-x64'
-        run: |
-          case ${{ matrix.target }} in
-            linux-arm)   TARGETARCH=armhf;;
-            linux-arm64) TARGETARCH=arm64;;
-          esac
-          dpkg --add-architecture ${TARGETARCH}
-          cat <<EOF > /etc/apt/sources.list.d/crossbuild.list
-          deb [arch=${TARGETARCH}] http://ports.ubuntu.com/ bionic main restricted
-          deb [arch=${TARGETARCH}] http://ports.ubuntu.com/ bionic-updates main restricted
-          deb [arch=${TARGETARCH}] http://ports.ubuntu.com/ bionic universe
-          deb [arch=${TARGETARCH}] http://ports.ubuntu.com/ bionic-updates universe
-          deb [arch=${TARGETARCH}] http://ports.ubuntu.com/ bionic multiverse
-          deb [arch=${TARGETARCH}] http://ports.ubuntu.com/ bionic-updates multiverse
-          deb [arch=${TARGETARCH}] http://ports.ubuntu.com/ bionic-backports main restricted universe multiverse
-          EOF
-          sed -Ei 's/^deb/deb [arch=amd64]/g' /etc/apt/sources.list
-          apt-get update
-          apt-get -y install libssl-dev:${TARGETARCH} zlib1g-dev:${TARGETARCH}
-
       - name: checkout workspace
         uses: actions/checkout@v3
         with:
           fetch-depth: 0
 
-      - name: install contemporary cmake
-        uses: lukka/get-cmake@latest
-
-      - uses: lukka/run-vcpkg@v10
-        with:
-          # use 2023.02.24 vcpkg baseline,
-          # see https://learn.microsoft.com/en-us/vcpkg/users/examples/versioning.getting-started#builtin-baseline
-          vcpkgGitCommitId: 'a7b6122f6b6504d16d96117336a0562693579933'
+      - name: Run CMake for Linux
+        if: ${{ matrix.container != null }}
+        run: |
+          ./ziti-builder.sh \
+            -p "ci-${{ matrix.preset }}" \
+            -t bundle
 
-      - uses: lukka/run-cmake@v10
-        name: Configure CMake
+      - name: Run CMake for non-Linux platforms
```

Comment:
I don't like that we have two different build flows -- Linux(script) vs non-Linux(action) -- they can get out of sync if steps are added to the build

Reply:
The uniformity of Linux CI and local Linux builds seems to be less important than I thought believed when I proposed `ziti-builder.sh`. Now there's more favor for uniformity of Linux and non-Linux in CI, so we should let this T-SDK build reflect the approach used in the C-SDK, a composite Action used by both Linux and non-Linux builds, and relocate the script in `./scripts/ziti-builder.sh` so that it can serve whomever who wants to build locally with the builder container, myself included. This will also solve caching and VCPKG baseline in the same way as C-SDK.